helper.go

```diff
@@ -181,6 +181,28 @@ func JsonMSet(client Client, ctx context.Context, kvs map[string]string, path st
 	return doMultiSet(client, ctx, cmds.s)
 }
 
+// DecodeSliceOfJSON is a helper that struct-scans each RedisMessage into dest, which must be a slice of pointer.
+func DecodeSliceOfJSON[T any](result RedisResult, dest *[]T) error {
+	values, err := result.ToArray()
+	if err != nil {
+		return err
+	}
+
+	ts := make([]T, len(values))
+	for i, v := range values {
+		var t T
+		if err = v.DecodeJSON(&t); err != nil {
+			if IsRedisNil(err) {
+				continue
```

Comment:
@rueian @sgasho If this happens, then `ts[i]` will be an empty value. Is this the expected behavior? Probably not.

Reply:
Ah, I see there is a test for this behavior, so probably it is a desired behavior. Sorry for the noise.